src/pages/patientView/clinicalInformation/lib/clinicalAttributesStyleConsts.js

```diff
@@ -0,0 +1,285 @@
+        formatter: t => {
+            return `${t} months`;
+        },
+        order: 10,
+    },
+    DRIVER_MUTATIONS: {
+        order: 6,
+    },
+    ERG_FUSION_ACGH: {
+        primary: true,
+        formatter: t => {
+            return `ERG-fusion aCGH: ${t}`;
+        },
+        order: 6,
+    },
+    ETS_RAF_SPINK1_STATUS: {
+        order: 6,
+    },
+    GLEASON: {
+        primary: true,
+        formatter: t => {
+            return `Gleason: ${t}`;
+        },
+    },
+    GLEASON_SCORE: {
+        order: 6,
+    },
+    GENDER: {
+        order: 1,
+    },
+    HISTOLOGY: {
+        order: 6,
+    },
+    KNOWN_MOLECULAR_CLASSIFIER: {
+        order: 5,
+    },
+    MOUSE_STRAIN: {
+        order: 6,
+    },
+    PATIENT_DISPLAY_NAME: {
+        primary: false,
+        order: 0,
+    },
+    OS_MONTHS: {
+        primary: false,
+        formatter: t => {
+            return `${t} months`;
+        },
+        order: 8,
+    },
+    OS_STATUS: {
+        primary: true,
+        values: {
+            DECEASED: '#f00',
+            DEAD: '#f00',
+            LIVING: 'rgb(0, 128, 0)',
+            ALIVE: 'rgb(0, 128, 0)',
+        },
+        order: 7,
+    },
+    SAMPLE_DISPLAY_NAME: {
+        primary: false,
+        order: 0,
+    },
+    SERUM_PSA: {
+        primary: true,
+        formatter: t => {
+            return `Serum PSA: ${t}`;
+        },
+        order: 6,
+    },
+    SEX: {
+        order: 1,
+    },
+    TMPRSS2_ERG_FUSION_STATUS: {
+        primary: true,
+        formatter: t => {
+            return `TMPRSS2-ERG Fusion: ${t}`;
+        },
+        order: 6,
+    },
+    TUMOR_GRADE: {
+        order: 6,
+    },
+    TUMOR_STAGE_2009: {
+        primary: true,
+        order: 6,
+    },
+};
+
+/**
+ * Functions for ordering and styling clinical attributes.
+ */
+/**
+ * Order patient clinical attributes by the object mapping config.
+ * If the key does not exist in config, then it shall be ordered last.
+ * @param {string} key
+ * @param {object} config
+ */
+export const getOrder = (key, config) => {
+    // Keys with lower numbers appear earlier
+    return key in config && 'order' in config[key]
+        ? config[key].order
+        : Number.MAX_VALUE;
+};
+
+/**
+ * Compare two keys a and b by their mapped ordering in ascending order.
+ * For instance, GENDER is displayed before OS_MONTHS.
+ * @param {string} a
+ * @param {string} b
+ * @param {object} config
+ */
+export const compare = (a, b, config) => {
+    return getOrder(a, config) - getOrder(b, config);
+};
+
+/**
+ * Builds a string to insert in an HTML element.
+ * Manually and conditionally display
+ * - leading comma on attribute after sample id
+ * - trailing comma on last displayed attribute
+ * on Patient row and Samples row. *
+ * @param {string} value
+ * @param {string} key
+ * @param {object} config
+ * @param {string[]} keys
+ */
+export const stringBuilder = (value, key, config, keys) => {
+    var builder = [];
```

Comment:
It'd be cool to mess around with monads here, but let's not.

Reply:
how do you mean?

Reply:
yeah, Luke, what's monad!?    i think we should get this merged soon so lets hold off.  but i am interested in  what that is!

Reply:
Oh, this was a joke. Please do not solve this using monads.

But to quote Saunders Mac Lane:
> All told, a monad in X is just a monoid in the category of endofunctors of X, with product × replaced by composition of endofunctors and unit set by the identity endofunctor.

Reply:
aha, I know this meme (very infamously and ironically spouted by a classmate in our Structure of Higher Level languages class)